app/Factories/AlbumFactory.php

```diff
@@ -74,12 +75,12 @@ public function findBaseAlbumOrFail(string $albumId, bool $withRelations = true)
 		}
 
 		try {
-			return $albumQuery->findOrFail($albumId);
+			return $albumQuery->findOrFail($albumId); // @phpstan-ignore-line
```

Comment:
Can we add a comment here about what kind of diagnostics we are ignoring?

Reply:
Fixed in https://github.com/LycheeOrg/Lychee/pull/1336/commits/3817459b78f214d434e1cf8b6cb86820b4631eae.